frontend/src/components/appbar/AppBar.tsx

```diff
@@ -10,14 +10,15 @@ import MenuIcon from "@material-ui/icons/Menu";
 import { Theme } from "@material-ui/core/styles";
 import Toolbar from "@material-ui/core/Toolbar";
 import Typography from "@material-ui/core/Typography";
+import { UserLoginStatusContext } from "../contexts/contexts";
 import { useTheme } from "@material-ui/core/styles";
 
 const drawerWidth = LayoutConstants.DRAWER_WIDTH;
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
-    root: {
-      display: 'flex',
+    accountIcon: {
+      marginLeft: "auto"
```

Comment:
```suggestion
      marginLeft: "auto",
```

Very minor nit: should have a comma here for consistency throughout the file.